### ISSUE TITLE
Configurations serializing items with same beginnings

### ISSUE THREAD

**Issue by Eldblom** [kamsar/Unicorn#166]
I have two configurations: News and Newsletter - they serialize items from /sitecore/templates/features/news and /sitecore/templates/features/newsletter. When adding items to the Newsletter I get the error below.

To recreate:

1. Create a blank Sitecore
2. Create a configuration "test" which serializes "/sitecore/layout/test"
3. Verify that items can be created below "/sitecore/layout/test"
4. Create a new configuration "testmore" which serializes "/sitecore/layout/testmore"
5. Create the item "/sitecore/layout/testmore" and add an item below this new item.
6. Fix the bug... :)

![image](https://cloud.githubusercontent.com/assets/11038361/17682611/05884a06-6390-11e6-8837-f7eb037fd5a1.png)

**Comment by kamsar** [comment]
I was unable to reproduce this on 3.3 RC4.

Test configurations:

```
<configuration name="Project.Prototype.Tests" description="Integration tests and dev support content of Project.Prototype">
	<targetDataStore type="Rainbow.Storage.SerializationFileSystemDataStore, Rainbow" physicalRootPath="$(sourceFolder)\Project\Prototype\Items\Tests" useDataCache="false" />

	<predicate type="Unicorn.Predicates.SerializationPresetPredicate, Unicorn" singleInstance="true">
		<!-- Integration Tests -->
		<include name="Tests" database="master" path="/sitecore/content/Prototype/home/integration-tests" />
	</predicate>

	<dataProviderConfiguration enableTransparentSync="false" type="Unicorn.Data.DataProvider.DefaultUnicornDataProviderConfiguration, Unicorn" singleInstance="true" />
</configuration>

<configuration name="Project.Prototype.Tests2" description="Integration tests and dev support content of Project.Prototype">
	<targetDataStore type="Rainbow.Storage.SerializationFileSystemDataStore, Rainbow" physicalRootPath="$(sourceFolder)\Project\Prototype\Items\Tests21" useDataCache="false" />

	<predicate type="Unicorn.Predicates.SerializationPresetPredicate, Unicorn" singleInstance="true">
		<!-- Integration Tests -->
		<include name="Tests" database="master" path="/sitecore/content/Prototype/home/integration-teststwo" />
	</predicate>

	<dataProviderConfiguration enableTransparentSync="false" type="Unicorn.Data.DataProvider.DefaultUnicornDataProviderConfiguration, Unicorn" singleInstance="true" />
</configuration>
```

Testing was performed with and without transparent sync enabled. Both ways, children of the integration test items could be created and serialized without error.

**Comment by kamsar** [comment]
Closing this as probably fixed in 3.3.1, unable to reproduce. Please reopen if still an issue.
